test/unit/report_importer_test.rb

```diff
@@ -19,54 +19,4 @@ class ReportImporterTest < ActiveSupport::TestCase
     r = Report.import read_json_fixture('report-no-logs.json')
```

Comment:
I think it ought to declare a test version of the class, we can't keep testing a deprecated method, nor should we only test one particular subclass (else we don't know if the base class alone, used by plugins, is working).